images/scripts/update-cilium-envoy-image.sh

```diff
@@ -7,49 +7,16 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
If possible I'd like to keep this update script as is to support the dev workflow of updating the envoy image to the image of feature branch in the `cilium/proxy` repo without the need to copy & paste the repo, version and digest. AFAIU i would have to do this and copy into the `Makefile.values` to achieve the same after this PR changes?

Reply:
as per the below PR, it seems like there is no post upgrade target required for cilium-proxy image. I will revert the change here.

https://github.com/cilium/cilium/pull/33602